src/components/DialogSections/index.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Should be moved inside cozy-ui, see
+ * https://github.com/cozy/cozy-ui/issues/1290
```

Comment:
In the issue, you say "This will not be done via ModalSections, we prefer now to use Dialog APIs". 

So what should be moved in UI?

Reply:
I removed the comment as I think DialogSections should not be moved to cozy-ui. It's OK if it stays in Banks.